docs/content/en/docs/yaml-crd-ref/taskdefinition.md

```diff
@@ -67,10 +72,11 @@ spec:
   timeout: <duration-in-seconds>
 ```
 
-The API ref points to
+TODO: The API ref points to
 [Kubernetes doc](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.24/#duration-v1-meta)
 where I don't find a direct hit
 but timeouts seem to be measured in seconds.
+Please verify
```

Comment:
```suggestion
```

Reply:
I removed the TODO.  We can always add info later.